src/storage/datacore/restapi/mode/alertscount.pm

```diff
@@ -0,0 +1,178 @@
+#
```

Comment:
I think it's more common to call this kin of mode simply alerts.pm because it doesn't only count for alerts it will also do other things around alerts (now or maybe later).